core/config/envvar/envvar.go

```diff
@@ -0,0 +1,73 @@
+package envvar
+
+import (
+	"fmt"
+	"log"
+	"os"
+
+	"github.com/pkg/errors"
+
+	"github.com/smartcontractkit/chainlink/core/config/parse"
+)
+
+var RootDir = New("RootDir", parse.HomeDir)
```

Comment:
This pattern has the advantage of doing the reflection up front just once, instead of lazily each time.